src/encoded/upgrade/pipeline.py

```diff
@@ -69,3 +70,15 @@ def pipeline_8_9(value, system):
     # https://encodedcc.atlassian.net/browse/ENCD-3773
     if value.get('status') == 'active':
         value['status'] = 'released'
+
+
+@upgrade_step('pipeline', '9', '10')
+def pipeline_9_10(value, system):
+    # https://encodedcc.atlassian.net/browse/ENCD-4711
+    for a in value['assay_term_names']:
+        if re.match('single-nuclei ATAC-seq', a):
+            new_assay_term_name = a.replace('single-nuclei', 'single-nucleus')
```

Comment:
I guess regex match might be overkill here?

Reply:
Yes, I am now using string equality instead.